src/python/epicscorelibs/test/test_ioc_ca.py

```diff
@@ -0,0 +1,97 @@
+import subprocess
+import os
+import sys
+import ctypes
+import tempfile
+import random
+import string
+
+try:
+    from queue import Queue
+except ImportError:
+    from Queue import Queue
+
+from epicscorelibs.ca import cadef, dbr
+
+
+class Channel:
+    def __init__(self, name):
+        self.name = name
+        self.chid = ctypes.c_void_p()
+
+    @property
+    def _as_parameter_(self):
+        # Setting this allows a channel object to autoconvert into the chid
+        # when passed to ca_ functions.
+        return self.chid.value
+
+
+def test_running_ioc():
+    if os.environ.get("PYTAG", "") == "win_amd64" \
+            and sys.version_info[:2] == (3, 4):
+        # This particular combination is broken. All the callbacks seem to
+        # return junk. Not sure why
+        return
```

Comment:
FYI. `raise unittest.SkipTest()` gives a bit more information to the test runner in this kind of situation.  I'm ok with dropping py3.4 windows builds entirely in this case.